@here/harp-mapview/lib/DataSource.ts

```diff
@@ -329,6 +329,19 @@ export abstract class DataSource extends THREE.EventDispatcher {
         return tileKey.level === zoomLevel;
     }
 
+    /**
+     * Returns `true` if [[MapView]] should traverse tiles further with given [[TileKey]] and
+     * zoom level.
+     *
+     * @param zoomLevel The zoom level of the [[MapView]].
+     * @param tileKey The unique identifier for a map tile.
+     * @returns `true` if the subtiles of the given [[TileKey]] should be
+     * checked for collisions.
+     */
+    shouldSubdivide(zoomLevel: number, tileKey: TileKey): boolean {
```

Comment:
I don't think this method is necessary in DataSource, it is fine enough to have it in the FrustumIntersection

Reply:
The idea here is to allow datasources to skip tile subdivision, defined by datasource's own rules